src/edu/csus/ecs/pc2/core/execute/JudgementUtilites.java

```diff
@@ -196,7 +196,9 @@ public static void dumpJudgementResultsToLog(Log log, ClientId judgeId, Run run,
 
             int numberOfTestCases = problem.getNumberTestCases();
             log.info(prefixString + " Dumping for " + run + " judge id=" + judgeId);
-            log.info(prefixString + " There are " + numberOfTestCases + " for problem " + problem.getDisplayName() + " '" + problem.getShortName() + "'");
+            String verb = numberOfTestCases > 1 ? " are " : " is " ;
+            String pluralform = numberOfTestCases > 1 ? " testcases " : " testcase ";
```

Comment:
For future pluralizing should use the Pluralize.pluralize method.

Reply:
Huh.  Didn't know (or at least, didn't remember) that class existed.  Nice.